### PR TITLE
Release v0.3.0

### PR DESCRIPTION
**Pull request by tsenart**
#### CHANGELOG
- Support Mesos 0.25 NetworkInfo as an IPSource (#266, #267, #268)
- Find home directory with environment variables (#275, #277, #278)

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,7 @@
+2015-09-23: v0.3.0
+  Support Mesos 0.25 NetworkInfo as an IPSource (#266, #267, #268)
```
Comment (sttts) [inline CHANGELOG]: It's not really clear to me what this means. "Add Mesos 0.25 support including NetworkInfo" ?

Comment (tsenart) [inline CHANGELOG]: There might be other Mesos 0.25 things that we still need to do so we can't say that.

Comment (sttts) [inline CHANGELOG]: Then make it clean in the changelog.

Comment (tsenart) [inline CHANGELOG]: ?

Comment (tsenart) [inline CHANGELOG]: Isn't it already saying exactly what feature from Mesos 0.25 we added support for?

Comment (sttts) [inline CHANGELOG]: People will read "0.25" is supported. Write something like "Experimental support ....".

Comment (tsenart) [inline CHANGELOG]: I'll rephrase it.

**Pull request merged**